model_analyzer/triton/model/model_config.py

```diff
@@ -258,14 +258,56 @@ def set_cpu_only(self, cpu_only):
 
     def cpu_only(self):
         """
-        Return
+        Returns
         -------
         bool
             Whether the model should be run on CPU only
         """
 
         return self._cpu_only
 
+    def is_ensemble(self) -> bool:
+        """
+        Returns
+        -------
+        bool
+           True if this is an ensemble model
+        """
+
+        return getattr(self._model_config, "platform") == "ensemble"
+
+    def get_ensemble_submodels(self) -> Optional[List[str]]:
+        """
+        Returns
+        -------
+            List[str]: Sub-model names
+        """
+
+        if not self.is_ensemble():
+            return None
+
+        submodels = [
+            model['modelName']
+            for model in self.to_dict()['ensembleScheduling']['step']
```

Comment:
Should we check the subkeys before indexing in? Seems to assume everything is in a correct state